user_guide_src/source/libraries/files/016.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$files->retainMultiplePatterns(['*.css*', '*.js']); // Would keep only *.css and *.js files
```

Comment:
```suggestion
$files->retainMultiplePatterns(['*.css', '*.js']); // Would keep only *.css and *.js files
```
Either the code or the comment is incorrect.

Reply:
That indeed is a typo. Thank you for catching it.